GiniVision/Assets/Localizable.strings

```diff
@@ -98,5 +98,6 @@
 "ginivision.imagepicker.openbutton" = "Öffnen";
 
 "ginivision.multipagereview.reorderContainerTooltipMessage" = "Die Seiten müssen in der richtigen Reihenfolge sein. Hier können Sie sie sortieren";
-
-
+"ginivision.multipagereview.addButtonLabel" = "Fügen Sie dieser Rechnung eine neue Seite hinzu";
+"ginivision.multipagereview.dragAndDropTip" = "Drag & Drop zum Sortieren";
+"ginivision.multipagereview.title" = "%d von %d";
```

Comment:
I'm usually not a fan of sprintf formatting in strings files. And here you might just get away with localizing the string in the middle.

The only problem is that there might be languages where the format is different and the string is not in the middle. However, even this format might not be sufficient for all languages.

What I'm saying is that I'm not sure about this, but I don't know how to proceed. We are unlikely to encounter such languages any time soon.

Reply:
Localizing strings in iOS is a mess. Maybe when I create the enum for the strings an images, I can handle this.